slatedb/src/row_codec.rs

```diff
@@ -142,8 +156,8 @@ impl SstRowCodecV0 {
     }
 
     pub fn encode(&self, output: &mut Vec<u8>, row: &SstRowEntry) {
-        output.put_u16(row.key_prefix_len as u16);
-        output.put_u16(row.key_suffix.len() as u16);
+        output.put_u16(row.key_prefix_len.try_into().unwrap());
+        output.put_u16(row.key_suffix.len().try_into().unwrap());
```

Comment:
unwrap -> expect with message

Reply:
unless we can get rid of this change, since it has been validated in constructor (same applies to value cast down below)

Reply:
I opted to leave the struct variables as usize for developer ergonomics. Bytes and such all want usize, so we'd have to cast back in a bunch of places. I'll add expect.